features.yaml

```diff
@@ -4,28 +4,255 @@ features:
+       area: Traffic Management
+  - name: "Prometheus Integration"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Client and Server Telemetry Reporting"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Service Dashboard in Grafana"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Distributed Tracing"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Stackdriver Integration"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Distributed Tracing to Zipkin/Jaeger"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Trace Sampling"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Observability
+  - name: "Request Classification"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Observability
+  - name: "WebAssembly Extension"
+    level:
+     - checklist:
+       maturity: Alpha
+       nextExpectedPromotion:
+       area: Extensibility
+  - name: "Service-to-service Mutual TLS"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Kubernetes: Service Credential Distribution"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Certificate management on Ingress Gateway"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Pluggable Key/Cert Support for istio CA"
+    level:
+     - checklist:
+       maturity: Stable
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Authorization"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "End User (JWT) Authentication"
     level:
      - checklist:
        maturity: Beta
        nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Automatic mutual TLS"
+    level:
+     - checklist: features/auto_mtls.md
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "VM: Service Credential Distribution"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+  - name: "Mutual TLS Migration"
+    level:
+     - checklist:
+       maturity: Beta
+       nextExpectedPromotion:
+       area: Security and policy enforcement
+  - name: "Standalone Operator"
```

Comment:
Is this server side operator controller or istioctl?

Reply:
This is referring to the server side [istioctl operator](https://istio.io/latest/docs/setup/install/operator/)